stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -2123,6 +2123,10 @@ internal constructor(
         lastSyncAt: Date,
     ): Call<List<ChatEvent>> {
```

Comment:
I would also do the check here so that there are no error logs in our sample app when logging in with a new user for the first time:
```
    private suspend fun syncHistoryForCachedChannels() {
        logger.d { "[syncHistoryForCachedChannels] no args" }
        repos.cacheChannelConfigs()

        // Sync cached channels
        val cachedChannelsCids = repos.selectAllCids()
        syncHistory(cachedChannelsCids) <--- here
    }
```

Reply:
Added an additional check inside `syncHistory ` method.